stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -65,6 +67,8 @@
   private ChildNodeRemovedEvent mCachedChildNodeRemovedEvent;
   private ChildNodeInsertedEvent mCachedChildNodeInsertedEvent;
 
+  private HashMap<String, List<Integer>> mSearchResults;
```

Comment:
because this is read and written from different threads, we need to synchronize it, e.g.
```
private final Map<String, List<Integer>> mSearchResults = Collections.synchronizedMap(
    new HashMap<String, List<Integer>>());
```
And by assigning to it here instead of in the constructor we can tag it with 'final'.